src/main/java/com/toptrumps/core/card/Card.java

```diff
@@ -37,14 +39,23 @@ public Attribute getAttributeByName(String name) {
                 .findFirst()
                 .orElse(null);
     }
+    private void findHighestAttributes() {
+        Supplier<Attribute> highestAttribute = () -> Collections.max(attributes);
+
+        for (Attribute a : attributes){
```

Comment:
We've been using the ```filter``` method from the ```streams``` API for a while now. Why not stick to it?